test/surface_test.py

```diff
@@ -1388,7 +1388,29 @@ def todo_test_get_pitch(self):
         #
         # This value is not needed for normal Pygame usage.
 
-        self.fail()
+        # Test get_pitch() on several surfaces of varying size
+        sizes = ((2, 2), (7, 33), (33, 7), (2, 734), (734, 2), (734, 734))
+        for width, height in sizes:
```

Comment:
My main thought here is - bit depths. It should be possible to add them into your tuples and then the creation of Surfaces and different bit depths should affect the values returned via `get_pitch()`, though if it works right it won't affect your tests as they start with `get_buffer()` which should also be affected in the same way.